models/_sector/nft/trades/nft_ethereum_trades_beta_legacy.sql

```diff
@@ -0,0 +1,73 @@
+{{ config(
+    schema = 'nft_ethereum',
+    alias = alias('trades_beta',legacy_model=True),
+    partition_by = ['block_date'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['project','project_version','tx_hash','sub_tx_trade_id']
+    )
+}}
+-- (project, project_version, model)
+{% set base_models = [
+     ('archipelago',    'v1',   ref('archipelago_ethereum_base_trades_legacy'))
+    ,('superrare',    'v1',   ref('superrare_ethereum_base_trades_legacy'))
+    ,('foundation',    'v1',   ref('foundation_ethereum_base_trades'))
```

Comment:
Missing legacy

Reply:
Found it with `dbt ls --resource-type model --select spellbook._sector.nft.trades.nft_ethereum_trades_beta_legacy,tag:dunesql`